api/types/types_deprecated.go

```diff
@@ -194,3 +195,101 @@ type ImageImportSource image.ImportSource
 //
 // Deprecated: use [image.LoadResponse].
 type ImageLoadResponse = image.LoadResponse
+
+// ContainerJSONBase contains response of Engine API GET "/containers/{name:.*}/json"
+// for API version 1.18 and older.
+//
+// Deprecated: use [container.InspectResponse] or [container.Base]. It will be removed in the next release.
+type ContainerJSONBase = container.Base
+
+// ContainerJSON is the response for the GET "/containers/{name:.*}/json"
+// endpoint.
+//
+// Deprecated: use [container.InspectResponse]. It will be removed in the next release.
+type ContainerJSON = container.InspectResponse
+
+// ContainerNode stores information about the node that a container
+// is running on.  It's only used by the Docker Swarm standalone API.
+//
+// Deprecated: ContainerNode was used for the classic Docker Swarm standalone API. It will be removed in the next release.
+type ContainerNode = container.ContainerNode
```

Comment:
I need to add some nolint comments here;

```
api/types/types_deprecated.go:215:22: SA1019: container.ContainerNode is deprecated: ContainerNode was used for the classic Docker Swarm standalone API. It will be removed in the next release. (staticcheck)
type ContainerNode = container.ContainerNode
                     ^